packages/itwinui-css/src/menu/mixins.scss

```diff
@@ -187,3 +187,10 @@ $iui-active-outline: 1px solid var(--iui-color-border-accent);
     fill: var(--_iui-list-item-icon-fill, var(--iui-color-icon));
   }
 }
+
+@mixin iui-transfer-list {
```

Comment:
lets move this to `transfer-list.scss` and create a new class, e.g. `iui-transfer-list-listbox`

Reply:
Fixed

Reply:
`role=listbox` needs to be brought back. It is needed, just not for styling.
![](https://github.com/iTwin/iTwinUI/assets/9084735/77a30c73-350c-49ee-ada7-404c78396f00)

Reply:
Fixed